### DISCUSSION TITLE
Can't login to account

### DISCUSSION THREAD

**Discussion by thuongtruong109** [community/community#124042] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi all,
I have some difficult in login to my Github account. Yesterday, i lost my iphone and reset my OS laptop, so i can't login to Github account. Currently, i can only access via password and email, other methods like passkey, 2FA code, recovery code on my old device are lost. 
How to resolve this problem usecase
Thank you very much!

**Comment by Rishit-katiyar** [comment]
Hi there,

Since you mentioned that you can still access your account via password and email, that's a good starting point. Here's what you can do:

1. **Recovery through Email**: If you still have access to the email associated with your GitHub account, you can initiate the password reset process. Visit the GitHub login page and click on the "Forgot password?" link. Follow the prompts to reset your password using the email recovery option.

2. **Contact GitHub Support**: If you encounter any difficulties during the password reset process or if you're unable to access your email, reach out to GitHub support for assistance. They may be able to verify your identity through alternate means and help you regain access to your account.

3. **Update Security Settings**: Once you regain access to your account, it's essential to update your security settings. Consider enabling two-factor authentication (2FA) with a new device or recovery codes. This adds an extra layer of security to your account and helps prevent unauthorized access in the future.

4. **Backup Recovery Codes**: In the future, it's a good practice to store recovery codes in a secure location, separate from your primary devices. These codes can be invaluable in situations where you lose access to your usual authentication methods.
